module/buff/basic_buff.cpp

```diff
@@ -274,54 +270,55 @@ void Detector::bgrProcessing(const int& _my_color) {
     case uart::RED:
       fmt::print("[{}] Image pre-processing color: RED\n", idntifier_green);
 
-      cv::subtract(split_img_[2], split_img_[0], bin_img_color_);
+      /* my_color为红色，则处理红色的情况 */
+      /* 灰度图与RGB同样做红色处理 */
+      cv::subtract(split_img_[2], split_img_[0], bin_img_color_);  // r-b
 
+#ifndef RELEASE
       if (buff_config_.ctrl.IS_PARAM_ADJUSTMENT == 1) {
         std::string window_name = {"[basic_buff] bgrProcessing() -> trackbar"};
 
         cv::namedWindow(window_name);
-        cv::createTrackbar("GRAY_TH_RED:", window_name, &buff_config_.param.RED_BUFF_GRAY_TH, 255, nullptr);
-        cv::createTrackbar("COLOR_TH_RED:", window_name, &buff_config_.param.RED_BUFF_COLOR_TH, 255, nullptr);
+        cv::createTrackbar("GRAY_TH_RED:", window_name, &buff_config_.param.RED_BUFF_GRAY_TH, 255,
+                           nullptr);
+        cv::createTrackbar("COLOR_TH_RED:", window_name, &buff_config_.param.RED_BUFF_COLOR_TH, 255,
+                           nullptr);
 
         cv::imshow(window_name, trackbar_img_);
+        fmt::print("🧐 BGR红色预处理调参面板已打开 🧐\n", idntifier_green);
```

Comment:
相关的显示标志位已成为排错的特征标志，先暂时加入回来

Reply:
```suggestion
        fmt::print("🧐 BGR红色预处理调参面板已打开 🧐\n");
```
在这里，这样的写法是有问题的。建议加入 idntifier_yellow 或其它比较显眼的 idntifier 来完成这件事

Reply:
ok